### DISCUSSION TITLE
GitHub actions POC

### DISCUSSION THREAD

**Discussion by PriyankaRathod03** [community/community#44581] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

For my project requirements, I am working on GitHub Actions POC.
I need to implement and use GitHub actions for Test Script Automation.
I am going through the docs, but still I have a few questions about the implementation part.
It will great if someone from GitHub support team can connect to me on a call.

Thanking in advance.
Priyanka Rathod.

**Comment by gondaljutt** [comment]
I would also like someone from Github support team to connect to me on a call.
